db/migrate/20210406114840_update_defendant.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateDefendant < ActiveRecord::Migration[6.0]
+  def change
+    change_table :defendants do |t|
+      t.remove :courtProceedingsInitiated
+    end
+  end
+end
```

Comment:
Need to add:
proceedingsConcluded
isYouth

Reply:
To keep things more straightforward I've not added all the non-required fields. If needed they can be added in the future.